src/cmd/cmdsdformat.rb.in

```diff
@@ -206,21 +214,25 @@ class Cmd
           exit(Importer.cmdDescribe(options['describe']))
         elsif options.key?('print')
           snap_to_degrees = 0
-          if options['preserve_includes']
-            Importer.extern 'int cmdPrintPreserveIncludes(const char *)'
-            exit(Importer.cmdPrintPreserveIncludes(File.expand_path(options['print'])))
-          elsif options.key?('snap_to_degrees')
+          precision = -9999
```

Comment:
Initially, I set it as `-9999` since technically negative numbers in `setprecision` are allowed but silently ignored. I agree that this was a little confusing so I modified it to `0` and updated the `-h` docs to state the argument must be a positive integer: 855cb1d390c029dcdd11b1e0085348a057a7c44e